src/actors/rqgActorSheet.ts

```diff
@@ -694,6 +695,33 @@ export class RqgActorSheet extends ActorSheet<
       });
     });
 
+    this.form?.querySelectorAll("[data-reputation-roll]").forEach((el) => {
+      let clickCount = 0;
+      el.addEventListener("click", async (ev: Event) => {
+        clickCount = Math.max(clickCount, (ev as MouseEvent).detail);
+
+        if (clickCount >= 2) {
+          // @ts-ignore wait for foundry-vtt-types issue #1165 #1166
+          const speakerName = this.token?.name || this.actor.data.token.name;
+          await CharacteristicCard.roll(
+            "Reputation",
+            this.actor.data.data.background.reputation || 0,
```

Comment:
For some reason I always get 0% chance when doubleclicking to do a direct roll.

Reply:
I wasn't getting quite that behavior, but I gave ReputationCard a "directroll" method which sets up flags as if you just clicked Roll on the card without making any changes.  This works the same way as the RuneMagicCard.